test/mainnet/helpers/SetupWithdrawer.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BSD Clause-3
+pragma solidity ^0.8.24;
+
+import {Test} from "lib/forge-std/src/Test.sol";
+import {Withdrawer} from "src/withdraws/Withdrawer.sol";
+import {TransparentUpgradeableProxy as TUProxy} from "src/Common.sol";
```

Comment:
don't use the TUPProxy alias just call it explicitly TransparentUpgradeableProxy